src/Popover/Popover.jsx

```diff
@@ -225,14 +217,13 @@ class Popover extends Component {
    */
   _applyAutoPositionIfNeeded(relativeTargetPosition, targetEl) {
     const targetPosition = Dom.offset(targetEl);
-
     // Movethe target position up so it is not scrolling past the bottom
     if (targetPosition.top + targetPosition.height > window.innerHeight) {
       // Initialize new top position
       let newTop = relativeTargetPosition.top;
 
       // Subtract enough pixels to get it inside the bounds of the window
-      newTop -= (targetPosition.top + targetPosition.height) - window.innerHeight;
+      newTop -= (targetPosition.height) - window.innerHeight;
```

Comment:
Since relative is a prop, have you tested to see if this would work if this.props.relative is false?